xarray/backends/api.py

```diff
@@ -147,7 +147,8 @@ def _get_lock(engine, scheduler, format, path_or_file):
 def open_dataset(filename_or_obj, group=None, decode_cf=True,
                  mask_and_scale=True, decode_times=True, autoclose=False,
                  concat_characters=True, decode_coords=True, engine=None,
-                 chunks=None, lock=None, cache=None, drop_variables=None):
+                 chunks=None, lock=None, cache=None, drop_variables=None,
+                 backend_kwargs={}):
```

Comment:
We can add this new `backend_kwargs` argument, but (1) it should pass arguments to each backends (otherwise it would be surprising) and (2) we should add it to `open_dataarray()`, too.

Reply:
I have added it to open_dataarray. Are you asking me to implement the keyword treatment for all other backends? Or just implement the passing and leave it unused?